Makefile

```diff
@@ -46,7 +46,8 @@ lint:
 	$(LINT_CMD)
 
 test-local: .env-SERVICE_KEY lint
-	TF_ACC=1 go test -v $(TEST_ARGS) ./logdna
+	TF_ACC=1 go test -v $(TEST_ARGS) ./logdna -coverprofile $(COVERAGE_FILE)
+	go tool cover -html $(COVERAGE_FILE) -o $(COVERAGE_FILE).html
```

Comment:
is there a way to test local without running coverage?  Just curious.

Reply:
nah, I found myself always wanting to see the coverage when running tests locally, and it doesn't take much time. but we could make it optional.

Reply:
it's ok, just curious more than anything. I'm used to the node suite being significantly slower, but you're right that isn't much of a factor for Go